modules/woo-commerce/qwc-admin.php

```diff
@@ -414,3 +414,9 @@ function qtranxf_wc_admin_filters() {
 }
 
 qtranxf_wc_admin_filters();
+
+
+function qtranxf_wc_admin_add_term_slugs($add_slugs_hooks){
+    return array_merge($add_slugs_hooks,
+            array('woocommerce_after_add_attribute_fields'));
+}
```

Comment:
This is not called

Reply:
fixed, 47e425f76a21f14b8818d34425e19c6951964b91